tests/gaudi/l2/vllm_buildconfig.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: vllm-workload
+  namespace: gaudi-validation
+spec: {}
+---
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: vllm-workload
+  namespace: gaudi-validation
+spec:
+  triggers:
+    - type: "ConfigChange"
+    - type: "ImageChange"
+  runPolicy: "Serial"
+  source:
+    git:
+      uri: https://github.com/opendatahub-io/vllm.git
```

Comment:
After comparing
1. https://github.com/opendatahub-io/vllm.git - ODH fork vllM
2. https://github.com/vllm-project/vllm - vLLM upstream
3.https://github.com/HabanaAI/vllm-fork - Habana fork vLLM
I think currently we should start from use the 3. with the change in 1 (adding the ubi based docker file for RH OpenShift), and obviously the Intel are upstreaming from 3 to 2. So in the long run we will using 2. 
So I think we need to 1). submit a PR to adding the ubi based docker file for RH, and also add the RH 9.4 support into the documents, and then 2). using repo 3   3) I think the owner of 3 will also help to upstream the ubi based docker file and doc to 2. 4) after that we can switch to use 2 the upstream vLLM.
@vbedida79  any comments? :-)

Reply:
https://github.com/HabanaAI/vllm-fork/pull/190 PR for ubi from RH can be merged into vllm gaudi repo, we can use that directly. Currently we can use the ubi image by RH maintained in https://github.com/opendatahub-io/vllm.git from https://github.com/HabanaAI/vllm-fork